README.md

```diff
@@ -37,4 +37,41 @@ pipx install git+https://github.com/MeltanoLabs/meltano-map-transform.git@v0.0.1
 
 ### Meltano installation instructions
 
-_...coming soon..._
+1. Add this plugin to your Meltano project
+
+   ```console
+   meltano add mapper meltano-map-transformer
+   ```
+
+1. Install the plugin
+
+   ```console
+   meltano install
+   ```
+
+1. Configure the plugin
+
+   ```yaml
+   # meltano.yml
+   plugins:
+     mappers:
+     - name: meltano-map-transformer
+       pip_url: "git+https://github.com/MeltanoLabs/meltano-map-transform.git"
+       executable: meltano-map-transform
+       mappings:
+       - name: hash_email
+         config:
+           stream_maps:
+             customers:
+               email: null # drop the PII field from RECORD and SCHEMA messages
```

Comment:
I believe `null` config values are being pruned from the config set before execution. 

This might not work as expected as of now.

Reply:
That's bizarre... 🤔

I mean, it's great but also unexpected... 😅

Reply:
@aaronsteers and I just confirmed that `__else__: null` also works fine: https://github.com/MeltanoLabs/meltano-map-transform/actions/runs/3200496207.

Maybe https://github.com/meltano/sdk/issues/983 is no longer true?

Reply:
@edgarrmondragon - Thanks for logging. I wonder if anyone can still repro the old issue, and if so, perhaps there is some difference in configuration or dialect. 

It's also possible with the various config fixes from Ken and Cody, we may have inadvertently fixed this issue also.

cc @visch